src/otx/algorithms/classification/configs/configuration.yaml

```diff
@@ -275,6 +275,29 @@ learning_parameters:
       type: UI_RULES
     visible_in_ui: true
     warning: null
+  input_size:
+    affects_outcome_of: NONE
+    default_value: Default
+    description: You can reduce training and inference time by using small input size.
```

Comment:
```suggestion
    description: The input size of the given model could be configured to one of the predefined resolutions. Reduced training and inference time could be expected by using smaller input size. Defaults to per-model default resolution.
```

Reply:
No need to accept this suggestion directly, but would you decide explicitly and apply to all other yaml files? (py files might need update, too.

Reply:
I'm sorry but I don't know which file needs to be changed. I already changed "training_base.py". Are there any other files I need to change?

Reply:
I get it, thanks for the update!